src/testFixtures/java/org/opensearch/knn/TestUtils.java

```diff
@@ -181,6 +188,43 @@ public static double calculateRecallValue(List<List<String>> searchResults, List
         return sum / recalls.size();
     }
 
+    public static Map<SpaceType, BiFunction<float[], float[], Float>> knnScoringBySpaceType = ImmutableMap.of(
```

Comment:
why this is public? if it's class level constant please change the case of the variable name to match other constants.